content/en/docs/concepts/sdk-configuration/otlp-exporter-configuration.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "OTLP Exporter Configuration"
+description: >-
+ Environment variables for configuring your OTLP Exporter.
+weight: 2
+---
+
+## Endpoint Configuration
+
+The following environment variables let you configure an OTLP/gRPC or OTLP/HTTP
+endpoint for your traces, metrics, and logs.
+
+### `OTEL_EXPORTER_OTLP_ENDPOINT`
+
+A base endpoint URL for any signal type, with an optionall-specified port
+number. Helpful for when you're sending more than one signal to the same
+endpoint and want one environment variable to control the endpoint.
+
+**Default value:**
+
+* gRPC: `"http://localhost:4317"`
+* HTTP: `"http://localhost:4318"`
+
+**Example:**
+
+* gRPC: `export OTEL_EXPORTER_OTLP_ENDPOINT="my-api-endpoint:443"`
```

Comment:
```suggestion
* gRPC: `export OTEL_EXPORTER_OTLP_ENDPOINT="https://my-api-endpoint"`
```

AFAIK all (or most) of the languages require `http` or `https` prefix do determine if secure on insecure gRPC connection should be used